includes/Core/Authentication/Google_Proxy.php

```diff
@@ -175,11 +176,7 @@ public function permissions_url( Credentials $credentials, array $query_args = a
 	 * @return string Complete proxy URL.
 	 */
 	public function url( $path = '' ) {
-		if ( defined( 'GOOGLESITEKIT_PROXY_URL' ) && GOOGLESITEKIT_PROXY_URL ) {
-			$url = GOOGLESITEKIT_PROXY_URL;
-		} else {
-			$url = self::BASE_URL;
-		}
+		$url = defined( 'GOOGLESITEKIT_PROXY_URL' ) && self::STAGING_BASE_URL === GOOGLESITEKIT_PROXY_URL ? self::STAGING_BASE_URL : self::PRODUCTION_BASE_URL;
```

Comment:
Let's break it into multiple lines:
```suggestion
		$url = defined( 'GOOGLESITEKIT_PROXY_URL' ) && self::STAGING_BASE_URL === GOOGLESITEKIT_PROXY_URL
			? self::STAGING_BASE_URL
			: self::PRODUCTION_BASE_URL;
```